tests/test_mesh.py

```diff
@@ -292,3 +295,33 @@ def test_usd_parse(usd_filename):
         check_gs_textures(
             gs_glb_material.emissive_texture, gs_usd_material.emissive_texture, 0.0, material_name, "emissive"
         )
+
+
+@pytest.mark.skipif(sys.version_info[:2] != (3, 10), reason="USD Baking works only in Python == 3.10")
+@pytest.mark.parametrize(
+    "usd_file", ["usd/WoodenCrate/WoodenCrate_D1_1002.usda", "usd/franka_mocap_teleop/table_scene.usd"]
+)
+@pytest.mark.parametrize("backend", [gs.gpu])
+def test_usd_bake(usd_file):
+    asset_path = get_hf_assets(
+        num_retry=1,
+        local_dir="tmp",  # If not using local dir, the symbol link of snapshot download will cause errors.
```

Comment:
I don't understand your comment: "# If not using local dir, the symbol link of snapshot download will cause errors."

There is no case where specifying "local_dir" should be necessary. You must be doing something wrong and `get_hf_assets` is broken.